app/controllers/facility_user_reservations_controller.rb

```diff
@@ -44,6 +44,10 @@ def load_user
     @user = User.find(params[:user_id])
   end
 
+  def not_customer_cancelable?(reservation)
+    reservation.present? && !reservation.canceled_at? && !reservation.can_cancel?
```

Comment:
Is there a way we can invert this logic and make it make sense? There's a lot of negation going on.